orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -614,6 +614,21 @@ private static Stage findAncestor(Stage stage, Execution execution, Predicate<St
     return children;
   }
 
+  /**
+   * Gets all direct children of the current stage. This is not a recursive method and will return
+   * only the children in the first level of the stage.
+   */
+  public List<Stage> getStageDirectChildren() {
```

Comment:
For consistency with other similar public methods (`directAncestors` and `ancestors`, `allDownstreamStages`)
```suggestion
  public List<Stage> directChildren() {
```